test/parallel/test-debug-prompt.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const assert = require('assert');
+const common = require('../common');
+const spawn = require('child_process').spawn;
+
+const timeoutId = setTimeout(function() {
```

Comment:
Is the timeout needed? Couldn't we just let the test timeout and fail instead of introducing a timer?